packages/node_modules/@webex/redux-module-media/src/actions.js

```diff
@@ -226,28 +251,32 @@ export function listenForCalls(sparkInstance) {
  * @param {Object} data.options
  * @returns {Promise}
  */
-export function placeCall(sparkInstance, {
-  destination,
-  options = {audio: true, video: true}
-}) {
+export function placeCall(sparkInstance, {destination, options}) {
   return (dispatch) => {
     let call, id;
 
-    try {
-      call = sparkInstance.phone.dial(
-        destination,
-        Object.assign(
-          options,
-          {
-            offerOptions: {offerToReceiveVideo: true, offerToReceiveAudio: true}
-          }
-        )
-      );
-      ({call, id} = processCall({dispatch, call}));
-    }
-    catch (e) {
-      // Ignore, will be caught on call.on('error')
-    }
+    getSupportedDevice()
```

Comment:
Return this promise as part of your dispatch:
```suggestion
    return getSupportedDevice()
```

Reply:
done!